### PR TITLE
Broadcast local operations

### LINKED ISSUE #3355: Add node produced operations to `subscribe_new_operations`
In testnet 18, we introduced experimental support for WebSockets where `subscribe_new_operations` broadcast only received operations from other nodes. Produced operation from the node are missing.

### PR DESCRIPTION
**Pull request by aoudiamoncef**
* [x] document all added functions
* [x] try in sandbox /simulation/labnet
* [x] unit tests on the added/changed features
  * [x] make tests compile
  * [x] make tests pass 
* [x] add logs allowing easy debugging in case the changes caused problems
* [x] if the API has changed, update the API specification

closes #3355

### CODE REVIEW

File: massa-protocol-worker/src/protocol_worker.rs [hunk 1]
```diff
@@ -511,8 +511,17 @@ impl ProtocolWorker {
                 self.checked_operations
                     .extend(operation_ids.iter().copied());
 
-                // Announce operations to active nodes not knowing about it.
                 let to_announce: Vec<OperationId> = operation_ids.iter().copied().collect();
+                // Broadcast operations to active sender(channel) subscribers.
+                if self.config.broadcast_enabled {
```
Comment (AurelienFT) [inline massa-protocol-worker/src/protocol_worker.rs:516]: In this function we receive operationids from the others nodes to let us know them and we ask their content if we don't already have them later on on the code.
So here you will fetch in the storage only ops that we already have and so has already been sent to the WS when the content has been retrieved.

Now that you want to broadcast all the operations you can retrieve the information for the `operation_pool` `add_operations` function ([code](https://github.com/massalabs/massa/blob/main/massa-pool-worker/src/operation_pool.rs#L101)). It's called by the `node_operations_from_node` where you had your previous send and the API when someone use `send_operations`

**Review by AurelienFT — CHANGES_REQUESTED** [review]

File: massa-protocol-worker/src/protocol_worker.rs (hunk 1 above)
Comment (aoudiamoncef) [reply massa-protocol-worker/src/protocol_worker.rs:516]: If I understand, we have to options:

1. Move the broadcast to add_operations in massa-pool-worker(I prefer this)
2. Broadcast from Public API.

Comment (aoudiamoncef) [reply massa-protocol-worker/src/protocol_worker.rs]: Done

**Review by AurelienFT — APPROVED** [review]
Nice one

**Pull request merged**